ui/editor/components/bubble-menu/ai.tsx

```
@@ -0,0 +1,47 @@
+import LoadingCircle from "@/ui/shared/loading-circle";
+import LoadingDots from "@/ui/shared/loading-dots";
+import Magic from "@/ui/shared/magic";
+import { Editor } from "@tiptap/core";
+import { BubbleMenu } from "@tiptap/react";
+import { useCompletion } from "ai/react";
+
+type Props = {
+  editor: Editor;
+};
+
+const AIBubbleMenu: React.FC<Props> = ({ editor }: Props) => {
```

ESLint

[error] 'React' is not defined.

(no-undef)